helpers/authentication.js

```diff
@@ -185,7 +185,7 @@ const login = (payload = {}, req, res, next) => {
 	}).catch((e) => {
 		res.locals.notification = {
 			type: 'danger',
-			message: 'Login fehlgeschlagen.',
+			message: res.$t("login.text.loginFailed"),
```

Comment:
```suggestion
			message: res.$t('login.text.loginFailed'),
```
strings must use single quotes